### DISCUSSION TITLE
New changes on the website... sounds exciting... will it change anything on the api side though?

### DISCUSSION THREAD

**Discussion by Braziliandre** [cloudbet/docs#88] [General]
Today, When I logged into the website I saw a pop up mmessage saying something about new changes to the website (the branding perhaps?) coming up on the 13th of this month. I was just curious if the APIs will be affected at all. We've been working on quite a significanlty involving bot project and just want to make sure things are good! 😅

**Comment by kgravenreuth** [comment]
Hi @Braziliandre ,
Yes! Cloudbet has a [brand new look](https://x.com/Cloudbet/status/1823210494356005131)

Cloudbet API works as before, glad to hear that you're using us in your project! Looking forward to tons of bets from you.

Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Reply by Braziliandre** [reply]
@kgravenreuth Love it! Always nice to try a new look every now and then! 😃 
We've been placing a decent amount already... (with monopoly money 😆). 
Still have a few things to iron out, but it's looking good so far. Thank you so much for the prompt assistance. 
I will certainly be back for more!
